.github/workflows/woodwork_main_airflow_performance_tests.yaml

```diff
@@ -0,0 +1,50 @@
+name: Run looking glass performance tests via Airflow
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  performance_tests:
+    name: Run looking glass performance tests via airflow
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: Generate ISO timestamp
+        run: |
+          echo "TIMESTAMP=$(date +"%Y-%m-%dT%H:%M:%S.%3NZ")" >> $GITHUB_ENV
+      - name: Checkout woodwork
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Get commit hashes
+        id: get_hashes
+        run: |
+          echo "CURRENT_HASH=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+          echo "Latest commit hash: ${{ env.CURRENT_HASH }}"
+          echo "PREVIOUS_HASH=$(git rev-parse --short HEAD~1)" >> $GITHUB_ENV
+          echo "Previous commit hash: ${{ env.PREVIOUS_HASH }}"
+      - name: Run airflow tests and generate report
+        run: | 
+          curl --location --request POST '${{ secrets.AIRFLOW_BASE_URL }}dags/woodwork_run_tests_generate_report/dagRuns' \
+          -u '${{ secrets.AIRFLOW_WW_USER }}:${{ secrets.AIRFLOW_WW_PASS }}' \
+          --header 'Content-Type: application/json' \
+          --data-raw '{
+            "conf": {
+                  "description": null,
+                  "lg_branch": "lg_baton_airflow",
+                  "n_trials": 1,
+                  "pytest_args": {},
+                  "python_version": "3.8",
```

Comment:
Is there any benefit to using a more recent Python version here?

Reply:
that sets the python version on which we run the woodwork tests. New LG has support for 3.8, 3.9, 3.10, and 3.11. We were previously testing on 3.8, but if all ww deps are compatible we can increase the version. Might need to do some testing to verify first, then update the payload as needed.